app/assets/stylesheets/style.css.scss

```diff
@@ -111,19 +149,29 @@ fieldset.tagline {
 }
 
 .rating {
-  border: .15em solid #41A5B2;
-  padding: .7em;
-  margin: .8em;
-  -moz-border-radius: 5px;
-	-webkit-border-radius: 5px;
+  border-bottom: 0.15em solid #41A5B2;
```

Comment:
Color literals like `#41A5B2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#41A5B2` should be written as `#41a5b2`